groupsforsections.py

```diff
@@ -255,7 +283,25 @@ def main():
 
         # mapping all the users in a sections to corresponding group
         groups_to_users_dict[group_id] = users
-        # TODO: Adding users to corresponding groups
+
+    failed_groups_to_users_dict={}
+    
+    # adding users to the group
+    for group in groups_to_users_dict:
+        for user in groups_to_users_dict[group]:
+            membership_id = add_users_to_group(groups_for_section_class, group, user)
+            if membership_id is None:
+                logging.error('The user %s is not added to the group %s' % (user, group))
```

Comment:
If api call return an error response, the details will be logged, in case of exception stack trace will have the details. Either case this will end up here with the `None` check and i am leaving a final message for clarity what happened